Controller/ImagineController.php

```diff
@@ -82,7 +81,6 @@ public function __construct(
      */
     public function filterAction(Request $request, $path, $filter)
     {
-        $path = PathHelper::urlPathToFilePath($path);
```

Comment:
Could you please describe the case when it is not required to convert URL into File system path? 
It would be great to cover it with test.

Reply:
If you look into the `urlPathToFilePath` method you will see this is only a `urldecode` applied to the `$path`.
But at this point, `$path` is already decoded when received in `filterAction`.